### PR TITLE
Add clarifying comment

### PR DESCRIPTION
**Pull request by phd3**
## Description

> Is this change a fix, improvement, new feature, refactoring, or other?

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

> How would you describe this change to a non-technical end user or system administrator?

## Documentation

(x) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Comment by bitsondatadev** [comment]
:wave: @phd3 - this PR has become inactive. If you're still interested in working on it, please let us know.

We're working on closing out old and inactive PRs, so if you're too busy or this has too many merge conflicts to be worth picking back up, we'll be making another pass to close it out in a few weeks.

**Comment by phd3** [comment]
@electrum could you please take a look ?

**Comment by colebow** [comment]
@phd3 do you think there's anyone else that could review this?

File: plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/HdfsInputFile.java
Comment (Praveen2112) [inline plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/HdfsInputFile.java:1]: @phd3 Does this comment in valid as of now, with the recent changes in code ?

Comment (phd3) [reply plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/HdfsInputFile.java:1]: hmm - this likely applies the same way to 

```
return new ForwardingSeekableInputStream(inputFile.newStream());
```

in `ForwardingInputFile` - but let me take a closer look at the new abstractions before rebasing.

**Comment by mosabua** [comment]
Can we close this @phd3 ?

**Comment by mosabua** [comment]
Closing this one out due to inactivity, but please reopen if you would like to pick this back up.

**Pull request closed without merging**